kura/distrib/RELEASE_NOTES.txt

```diff
@@ -1,61 +1,13 @@
-Eclipse Kura - 5.3.0 - May 2023
+Eclipse Kura - 5.3.1 - June 2023
 -------------------------------------------------------------------------------------------------
 Description:
-This minor release of Eclipse Kura, compatible with Java 8 and OSGi R7, introduces a new framework compatibility with Java 17, 
-a completely renewed networking that leverages Linux NetworkManager, a new set of APIs and abstractions for SQL and noSQL databases, 
-the availability of SQLite as a database option (H2 still available for compatibility reasons), 
-an improved Container Management feature with UDP ports support, a refreshed Asset implementation with the new data publishing on change and, finally, 
-the new Kura Data Service Connection Policy.
+This service release of Eclipse Kura includes the following changes:
+- Fixed metatype issue in Wires Conditional component preventing correct displaying in the Web UI
+- Fixed distribution issue with SQLite dp
```

Comment:
Is it worth to mention this? It has no impact for the end user. If we want to mention this we should mention also the wires script tools dp which had the same issue.